media/css/mozorg/home/takeover.less

```diff
@@ -0,0 +1,223 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../../sandstone/lib.less";
+
+@mozRed: #c33d32;
+@backgroundColor: #f89c24;
+@foregroundColor: #fff;
+@optionInactive: #90939d;
+@optionActive: #259BEB;
+
+.fundraising-takeover * {
+    .border-box;
+}
+
+.fundraising-takeover {
+    position: absolute;
+    top: 0;
+    left: 0;
+    background: @backgroundColor url('/media/img/home/takeover/hart.svg') center top no-repeat;
+    .background-size(100%);
+    color: @foregroundColor;
+    width: 100%;
+    .border-box;
+    z-index: 999;
+
+    @media only screen and (max-width: @breakDesktopWide) {
+        .background-size(1500px);
+        background-position: center 0;
+    }
+
+    @media only screen and (max-width: @breakTablet) {
+        .background-size(1000px);
+        background-position: center 0;
+        text-align: center;
+    }
+
+    &:after {
+        position: absolute;
+        bottom: 0;
+        left: 55%;
+        content: '';
+        background: transparent url('/media/img/home/takeover/mrfox.svg') 0 0 no-repeat;
+        width: 464px;
+        height: 550px;
+
+        @media only screen and (max-width: @breakDesktop) {
+            width: 380px;
+            height: 450px;
+        }
+
+        @media only screen and (max-width: @breakTablet) {
+            display: none;
+            background: none;
+        }
+    }
+
+    .moz-logo {
+        width: 276px;
+        height: 93px;
+
+        @media only screen and (max-width: @breakDesktop) {
+            width: 200px;
+            height: 67px;
+        }
+    }
+
+    h2 {
+        .font-size(3.5rem);
+        line-height: 1.2;
+
+        @media only screen and (max-width: @breakDesktop) {
+            .font-size(2.5rem);
+        }
+
+        @media only screen and (max-width: @breakTablet) {
+            .font-size(2rem);
+        }
+
+        @media only screen and (max-width: @breakMobile) {
+            .font-size(1.5rem);
+        }
+    }
+
+    .header-strip {
+        background-color: #fff;
+        padding: 10px;
+        font-weight: bold;
+        text-transform: uppercase;
+
+        a:link,
+        a:visited,
+        a:hover {
+            color: @mozRed;
+        }
+    }
+
+    .contained {
+        margin: 0 auto;
+        padding: 0 (@baseLine * 3);
+        max-width: 960px;
+
+        @media only screen and (max-width: @breakTablet) {
+            padding: 0 10px;
+        }
+    }
+
+    .copy {
+        margin-top: (@baseLine * 6);
+        padding-bottom: (@baseLine * 2);
+
+        @media only screen and (max-width: @breakTablet) {
+            margin-top: (@baseLine * 3);
+        }
+
+        @media only screen and (max-width: 360px) {
+            margin-top: @baseLine;
+        }
+
+        header {
+            max-width: 530px;
```

Comment:
This style is applied for all breakpoints, breaking smaller layouts. We could just use `width` here instead of `max-width` and set an value for each breakpoint to make sure things reflow nicely.